packages/libs/react-ui/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  boldTextClass,
+  imageClass,
+  imageContainerClass,
+  linkClass,
+  linkContainerClass,
+  profileCardClass,
+  tagClass,
+  tagContainerClass,
+} from './ProfileCard.css';
+
+import { Grid } from '@components/Grid';
+import { Tag } from '@components/Tag';
+import React, { FC } from 'react';
+
+export default {};
+export interface IProfileCardProps {
+  name: string;
+  title: string;
+  imageSrc: string;
+  tags?: string[];
+  links?: Record<string, string>;
+}
+
+export const ProfileCard: FC<IProfileCardProps> = ({
+  name,
+  title,
+  imageSrc,
+  tags = [],
+  links = {},
+}) => {
+  return (
+    <div className={profileCardClass}>
+      <Grid.Root columns={12}>
+        <Grid.Item columnSpan={2}>
+          <div className={imageContainerClass}>
+            <img className={imageClass} src={imageSrc} alt={name} />
+          </div>
+        </Grid.Item>
+        <Grid.Item columnSpan={10}>
+          <p className={boldTextClass}>{name}</p>
+          <p>{title}</p>
+          <ul className={tagContainerClass}>
```

Comment:
you should only render these lists if they have children, otherwise you're cluttering the dom with empty elements